config/prow/plugins.yaml

```diff
@@ -454,6 +454,11 @@ project_config:
               To do
             Doc Writing and Editing:
               To do
+    kubernetes-sigs:
+      org_maintainers_team_id: 3162587 # sig-testing-dummy-project-team
```

Comment:
Is this right? I copied it from the `kubernetes` org above. Could I instead leave it off?

Reply:
let's use the `2842373` which is the id for the github admin team. ( see https://github.com/kubernetes/test-infra/blob/75e6b24b1b7afebfdddaba9104da55f5df66f288/config/prow/plugins.yaml#L417-L419 )

Reply:
Come to think of it, shouldn't the team need to exist on the `kuberentes-sigs` org? I can't see a "Github Admin Team" on either org (presumably it is secret?) but I would assume the ID would be different even if an equivalent exists both places. No admin team for the SIGs org is mentioned in this file AFAICT. At a guess, maybe it is generated from [this part of the org file](https://github.com/kubernetes/org/blob/20eea7174db48d98eec676c8f8b15bd210c19ce4/config/kubernetes-sigs/org.yaml#L8-L17).  But how do I find the ID?